Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/sheets/person/ContactDiaryPersonBottomSheetDialogViewModel.kt

```diff
@@ -21,22 +21,33 @@ class ContactDiaryPersonBottomSheetDialogViewModel @AssistedInject constructor(
     @Assisted private val addedAt: String?,
     private val contactDiaryRepository: ContactDiaryRepository
 ) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
+
     private val text = MutableStateFlow("")
 
     val isValid = text.map {
-        it.isNotEmpty() && it.length <= MAX_PERSON_NAME_LENGTH
```

Comment:
Could still be kept IMHO. It's nicer to have a warning that the text is too long (even if white spaces are trimmed afterward) than have text cut off silently.

Reply:
There is not warning. It just disables the save button.

Reply:
And the indicator, that more than 250 chars have been reached in red below the text field

Reply:
<img width="407" alt="Screenshot 2020-12-21 at 14 00 56" src="https://user-images.githubusercontent.com/56824/102779687-f3d55f80-4394-11eb-99af-1166a86a8c7f.png">

Reply:
I put it back in